providers/cloudns/api.go

```diff
@@ -233,5 +235,4 @@ func fixTTL(ttl uint32) uint32 {
 		}
 	}
 
-	return allowedTTLValues[0]
-}
+	return allowedTTLValues[0]}
```

Comment:
This change seems unnecessary. Please run `gofmt -s -w api.go`

Reply:
done